packages/components/src/button/index.native.js

```diff
@@ -12,7 +12,7 @@ export default function Button( props ) {
 			onPress={ onClick }
 			style={ { borderColor: ariaPressed ? 'black' : 'white', borderWidth: 1, borderRadius: 2 } }
 		>
-			<View style={ { flex: 1, flexDirection: 'row' } }>
+			<View style={ { height: 44, width: 44, flexDirection: 'row' } }>
```

Comment:
Perhaps you've already tried it but I wonder, can we just set the height of the toolbar hosting the buttons instead of the button itself? I'm thinking that it might be easier to maintain if we only constrain a single element (toolbar) instead of the various elements that might end up being inside the toolbar.

Reply:
@hypest thanks for the feedback. This is just a temporary solution on this PR as we don't have toolbar and button style at all. (With flex: 1 we had a problem as buttons weren't visible, so I just changed to 44 size) I have a separate task for implementing a style into buttons and toolbar which will have new PR. Is this sounds reasonable for you?

Reply:
> Is this sounds reasonable for you?

Perfectly, thanks!